client/js/components/user/CustomerSettings/CustomerSettingsBranding.vue

```diff
@@ -47,12 +48,12 @@
           style="max-width: 300px;">
         <dp-checkbox
           id="r_logoDelete"
+          v-model="logoDelete"
```

Comment:
most of the time we are using something like `isAnything` for boolean variable names so we can spot a boolean already by its variable name 

```suggestion
          v-model="isDeleteLogo"
```

The data prop name must be adjusted then, too.

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2212/commits/e09ca1f649d4ecf10ba9f280eba551c7c41900ed